packages/mdc-dom/index.js

```diff
@@ -21,6 +21,6 @@
  * THE SOFTWARE.
  */
 
-import * as ponyfill from './ponyfill';
```

Comment:
this file should be switched to a .ts file as well

Reply:
Done